src/systems/userinput/bindings/keyboard-mouse-user.js

```diff
@@ -68,6 +69,16 @@ export const keyboardMouseUserBindings = addSetsToBindings({
       dest: { value: paths.actions.characterAcceleration },
       xform: xforms.normalize_vec2
     },
+    {
+      src: { value: paths.device.mouse.wheel },
+      dest: { value: dCharSpeed },
```

Comment:
Couldn't this just write directly to `paths.actions.dCharSpeed` instead of using an intermediate variable path?